### DISCUSSION TITLE
Re: "Creating Tensors" - Tensors and Strings/Characters?

### DISCUSSION THREAD

**Discussion by Shiva-Menon-97** [mrdbourke/pytorch-deep-learning#218] [Q&A]
Hey everyone,

Can a Tensor (be it a Scalar/Vector/Tensor) contain a character, or strings? Because when I tried them as shown below:

```
Scalar_Character = torch.tensor('p')
Scalar_String = torch.tensor("Hello")
```

I get this error : " ValueError: too many dimensions 'str' "

Is it that Pytorch Tensors can't hold characters or strings? 

If this is already explained in later sections of the tutorial, please do forgive me!

**Comment by Varzil** [comment]
https://stackoverflow.com/questions/65804689/with-bert-text-classification-valueerror-too-many-dimensions-str-error-occur
This might help.
